thanks-bot/bot.js

```diff
@@ -25,24 +24,23 @@ const ALLOWED_CHANNEL_ID = '525249065039036426'
 client.on('message', message => {
   const thanksMessageRegexp = /!thanks\b/gmi
 
-  // Conditions for exiting the function
+  // Ignoring the following cases
   if (message.channel.id !== ALLOWED_CHANNEL_ID ||
     message.author.id === client.user.id ||
-    message.content !== thanksMessageRegexp.test(message)) {
+    !thanksMessageRegexp.test(message)) {
     return
   }
 
-  // mentioned Users
   let mentionedUsers = message.mentions.users
 
   // Filtering mentionedUserIDs
-  let mentionedUserIDs = mentionedUsers.map(item => item.id)
+  // let mentionedUserIDs = mentionedUsers.filter(item => item.id !== message.author)
```

Comment:
Let's remove it. Even if we need in the future, it's a small piece of code and everyone should be able to write it.